### PR TITLE
(Refactor) Refactor RadioInputField and add tests

### LINKED ISSUE #442: (Feature) Make RadioInputField more generic (right now it can only have two options)
If you are reporting a problem with ICO Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

All

### If you were able to create it, what is the URL of your crowdsale?

No

### Do you have screenshots showing the problem?

No

### Do you see errors in the dev console? If yes, please include a screenshot

(To open the dev console in Google Chrome, press F12, or go to `View -> Developer -> Developer Tools`, and open the Console tab)

No

---

### PR DESCRIPTION
**Pull request by fvictorio**
Closes #442.

This PR makes RadioInputField acccept more than two options, and improves a little its code. It also adds tests for the component.

I thought about using it for the gas price selector, but that component is special because it shows an input when you select a custom gas price, so I ended up leaving it as it was.

### CODE REVIEW

File: src/components/stepThree/index.js [hunk 1]
```diff
@@ -364,7 +360,7 @@ export class stepThree extends React.Component {
             <div className="hidden">
               <div className="input-block-container">
                 <InputField
-                  side="left"
+                  extraClassName="left"
```
Comment (fernandomg) [inline src/components/stepThree/index.js]: it should be `side` for the `InputField`

Comment (fvictorio) [reply src/components/stepThree/index.js]: Oops, thanks. Fixed.

**Comment by fvictorio** [comment]
You mean how to run tests for a single component? You have to pass the name of the component to `jest`:

```
npm run test:dapp -- ComponentName
```

(Actually, I think that runs all the tests that match `ComponentName`, but it's good enough for most scenarios)

If that's what you meant, I'll add it to the wiki.

**Comment by vbaranov** [comment]
Yes, this is actually what I mean. It would be great if it would be documented in wiki.

**Pull request merged**